sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/autorest.md

```diff
@@ -13,9 +13,11 @@ batch:
   model-namespace: false
 
 # TODO: Uncomment when we ship authoring support and remove ./QuestionAnsweringClientOptions.cs.
-# - input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/34a2c0723155d134311419fd997925ce96b85bec/specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json
-#   add-credentials: true
-#   low-level-client: true
+- input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/34a2c0723155d134311419fd997925ce96b85bec/specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json
```

Comment:
I just noticed that line 13 above (`model-namespace: false`) has too many spaces. It still seems to work, but could you fix the spacing and generate again just to make sure everything is technically correct?

Reply:
Sorry but I don't seem to be able to find the extra spaces. Do you mean the indentation before `model-namespace: false`? Autorest does not generate without having two spaces before it like the rest of the properties. 

The first property for each swagger is the only one with a single space. i.e. `- input-file: ...`

Reply:
Hmm. When I looked at my local source I don't see extra spaces. I guess it's just a GitHub rendering issue. Never mind then.